test/unit/lookups/olleh_test.rb

```diff
@@ -248,7 +260,7 @@ def test_olleh_result_wgs_coordinates
   end
 
   def test_olleh_addr_nearest_position_search
-    VCR.use_cassette('nearest_position') do
+    VCR.use_cassette('address/nearest_position') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.